signac/core/synced_collections/synced_collection.py

```diff
@@ -128,10 +128,19 @@ class SyncedCollection(Collection):
     _validators: List[Callable] = []
     # Backends that support threading should modify this flag.
     _supports_threading: bool = False
+    _LoadSaveType = _LoadAndSave
 
     def __init__(self, parent=None, *args, **kwargs):
-        self._parent = parent
-        self._suspend_sync_ = 0
+        if parent is not None:
+            self._parent = parent._parent if parent._parent is not None else parent
+        else:
+            self._parent = parent
+        self._suspend_sync = (
+            _CounterContext() if parent is None else parent._suspend_sync
+        )
+        self._load_and_save = (
+            type(self)._LoadSaveType(self) if parent is None else parent._load_and_save
+        )
```

Comment:
It might make more sense to get these attributes from `self._root` instead of the passed `parent` (depending on the suggested name change).